.github/workflows/test_generator_backend.yml

```
@@ -0,0 +1,105 @@
+        id: find_related_files
+        run: |
+          cp tools/test_generator/scripts/find_related_files.py ./find_related_files.py
+          
+          chmod +x find_related_files.py
+          
+          python find_related_files.py backend_relevant_files.txt $GITHUB_WORKSPACE
+          
+          if [ -s all_files.txt ]; then
+            echo "===== Original files with their dependancies ====="
+            cat dependencies_map.json | jq
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)